docs/source/remote_access.rst

```diff
@@ -0,0 +1,80 @@
+
+Now from a terminal window on your **local machine**, you can try logging in to ``hostalias`` with the following command: ``$ ssh -i ~/.ssh/id_rsa_hostalias hostalias``. This will prompt you for the password you specified upon creating your key pair using ``ssh-keygen``. To always make use of your private key when logging in to ``hostalias``, add the following to your ``config`` file on your **local machine**::
+
+   Hostname **hostname**
+      User **username**
+      LocalForward 3039 localhost:3039
+      IdentityFile **~/.ssh/id_rsa_hostname**
+      UseKeychain yes
+      AddKeysToAgent Yes
+
+You should not be able to log in simply by typing ``$ ssh hostalias``. Congratulations! 
+
+
+Edit remote files in a local text editor using ``rmate``
+--------------------------------------------------------
+As an alternative to remote-based text editors such as ``vi`` and ``emacs``, we can
+use port-fowarding to set up a `local-based text editor like Atom <https://atom.io>`_ which includes features such as syntax highlighting and code completion. For instructions to install ``rmate`` on your **local machine**, `click here <https://github.com/textmate/rmate>`_. Then to specifically use Atom to edit remote files, `click here <https://atom.io/packages/remote-atom>`_. You will need to add the following line to your ``~/.ssh/config`` file: ``RemoteForward 52698 localhost:52698``.
+
+
+Edit remote python files in a ``jupyter`` environment
+-----------------------------------------------------
+The ``jupyter`` environment is a great environent for data exploration and integrating
+your figures inline with your code. To open your first Jupyter notebook, log in to your **remote machine** and type: ``$ jupyter lab --no-browser --port=3039``. This should function because of all the work we put in during the port forwarding section. To shorten this command, add an alias to your ``~/.bashrc`` file on your **remote machine**. I personally use the alias ``rjlab``
+
+Recap
+-----
+Your final ``~/.ssh/config`` file should look like this (making sure to replace the text surrounded by ``**double asterisks**`` with your own information)::
+
+   Host **hostalias**
+      Hostname **hostname**
+      User **username**
+
+   Hostname **hostname**
+      User **username**
+      LocalForward 3039 localhost:3039
+      RemoteForward 52698 localhost:52698
+      IdentityFile **~/.ssh/id_rsa_hostname**
+      UseKeychain yes
+      AddKeysToAgent Yes
+
+**One final note:** Remember the port numbers chosen are arbitary. If you choose the same number as someone else on your network, their files may open up on your computer and vice versa!
```

Comment:
Also add a note that if connection is interrupted you can still reconnect, assuming the jupyter process is still running. But make sure not to leave zombie jupyter processes with open ports on remote hosts!